k8s/cassandra/README.md

```diff
@@ -61,11 +61,10 @@ git clone --recursive https://github.com/GoogleCloudPlatform/click-to-deploy.git
 An Application resource is a collection of individual Kubernetes components,
 such as Services, Deployments, and so on, that you can manage as a group.
 
-To set up your cluster to understand Application resources, navigate to the
-`k8s/vendor` folder in the repository, and run the following command:
+To set up your cluster to understand Application resources, run the following command:
 
 ```shell
-kubectl apply -f marketplace-tools/crd/*
+kubectl apply -f click-to-deploy/k8s/vendor/marketplace-tools/crd/*
```

Comment:
We should have the `cd` command together with the clone command and drop `click-to-deploy`?

Reply:
+1 - other than that LGTM.

Reply:
Thanks.

The most important thing is that all instructions are clear. This may be a future request.